SpatialGDK/Source/SpatialGDKServices/Private/LocalDeploymentManager.cpp

```diff
@@ -242,21 +242,21 @@ bool FLocalDeploymentManager::TryStopLocalDeployment()
 	bool bSuccess = false;
 
 	TSharedPtr<FJsonObject> SpotJsonResult;
-	bool bPasingSuccess = FSpatialGDKServicesModule::ParseJson(SpotDeleteResult, SpotJsonResult);
-	if (!bPasingSuccess)
+	bool bParsingSuccess = FSpatialGDKServicesModule::ParseJson(SpotDeleteResult, SpotJsonResult);
```

Comment:
This was already fixed I believe. We need to merge release into master.

Reply:
Merged with master and still here :D